### ISSUE TITLE
How do tacotron partition training set and test set?

### ISSUE THREAD

**Issue by AeolusZane** [keithito/tacotron#328]
i found that all the .wav files are used to train. is there no test set?

**Comment by fathimad11** [comment]
I also have the same doubt..
